packages/form-js-editor/src/render/components/properties-panel/entries/ValuesSourceEntries.js

```diff
@@ -0,0 +1,219 @@
+import { get, isUndefined, without } from 'min-dash';
+
+import { SelectEntry, isSelectEntryEdited, TextFieldEntry } from '@bpmn-io/properties-panel';
+import ValueEntry from './ValueEntry';
+
+import { arrayAdd } from '../Util';
+import { useService } from '../../../hooks';
+
+export function ValuesSourceSelectorEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [ {
+    id: 'values-source',
+    component: ValuesSourceSelect,
+    isEdited: isSelectEntryEdited,
+    editField,
+    field,
+  } ];
+}
+
+export function InputKeyValuesSourceEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [{
+    id: 'input-values-key',
+    component: InputValuesKey,
+    label: 'Input values key',
+    description: 'Define which input property to populate the values from',
```

Comment:
I'm fine with this, but I'm also not an expert in technical wording. Maybe @christian-konrad can give some feedback on this.

![image](https://user-images.githubusercontent.com/9433996/177758582-36ac8e61-386c-48f8-92ca-5baa8f52e3ba.png)